I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 458, in script
    jump fnr_dialogue
ScriptError: could not find label 'fnr_dialogue'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 458, in script
    jump fnr_dialogue
  File "/Applications/renpy-8.3.2-sdk/renpy/ast.py", line 1699, in execute
    rv = renpy.game.script.lookup(target)
  File "/Applications/renpy-8.3.2-sdk/renpy/script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'fnr_dialogue'.

macOS-14.6.1-arm64-arm-64bit arm64
Ren'Py 8.3.2.24090902
Balancing the flow 1.0
Sat Nov 16 21:57:19 2024
